### ISSUE TITLE
⌨ Experience skill details text area not unique

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#11750]
## ⌨ Accessibility Issue

The text areas to fill ion details when connecting skills to an experience do not have unique labels.

### WCAG 2.1 Criteria

https://www.w3.org/WAI/WCAG22/Techniques/general/G131.html

## 🦋 Expected Behaviour

Form labels should be descriptivbe and clear about its purpose.

## 🕵️ Details

By having identical labels with no additional context as to the specific purpose of each, it may not be clear for the user.

## 📋 Steps to Reproduce

1. Login as any user
2. Go to create an experience
3. Add multiple skills
4. Confirm the details inputs all have the exact same label

## 📸 Screenshot

![2024-10-11_10-21](https://github.com/user-attachments/assets/4b6f05e5-cdf1-4f77-8239-e8b11c31a8b8)

## 🙋‍♀️ Proposed Solution

We have the skill name just above, we could:

1. Include it as an additional label
2. Add it as the real label for the input ("Describe how this skill featured in this role

## ✅ Acceptance Criteria

- [ ] Labels are unique and descriptive
